assets/check

```diff
@@ -60,7 +61,8 @@ fi
 # collect all pull requests from uri
 REMOTES=$(git ls-remote "$uri")
 set +e
-PULL_REQUESTS=$(echo "$REMOTES" | grep -E "/pull\\-requests/[0-9]+/${prq_branch}")
+PULL_REQUESTS=$(echo "$REMOTES" | grep -E "/pull\\-requests/[0-9]+/${prq_branch}" | \
+                sort -t / -k 3 -n -r | head -n $max_pr_to_check)
```

Comment:
Sort by the PR number and sort in reverse, taking the top `$max_pr_to_check` ensures we get the most recent ones.